app/components/Layout.tsx

```diff
@@ -66,6 +65,8 @@ const Layout: React.FC<Props> = ({ children, session, title }) => {
       </Head>
       <Navigation isLoggedIn={isLoggedIn} />
       <StyledMain>{children}</StyledMain>
+      <TimeTravel />
+
```

Comment:
This should be conditionally enabled, based on the `ENABLE_MOCK_TIME` setting

Reply:
Implemented